simple-wp-helpdesk/src/Rest/BulkController.php

```
@@ -174,8 +174,22 @@ protected function apply( string $action, int $id, array $params ): void {
 			case 'assign':
 				$raw_assignee = isset( $params['assignee'] ) ? $params['assignee'] : 0;
 				$assignee     = is_scalar( $raw_assignee ) ? (int) $raw_assignee : 0;
-				update_post_meta( $id, '_ticket_assigned', $assignee > 0 ? $assignee : '' );
+				$old_assigned = swh_get_int_meta( $id, '_ticket_assigned_to' );
+				update_post_meta( $id, '_ticket_assigned_to', $assignee > 0 ? $assignee : '' );
 				TicketVersion::bump( $id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\SWH\Concurrency\TicketVersion' in method 'apply'. (undefined)

(StaticAccess)